storybook-utilities/markdownDocumentationLinkBuilder.js

```diff
@@ -1,6 +1,7 @@
-const markdownDocumentationLinkBuilder = (type, component) => {
+const markdownDocumentationLinkBuilder = (component, type) => {
```

Comment:
Hey @guern1kn, thanks for making the suggested change but you could do with by setting a default parameter. ES6 allows you to define a default for a param if its undefined. See https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Functions/Default_parameters.  

```suggestion
const markdownDocumentationLinkBuilder = (component, type='components') => {
```
So you don't need the whole line `const directory = !type ? 'components' : type;`.

Doing it as part of the argument definition makes it clearer what the func is doing right when you read the available options.